_posts/2020-03-04-developer-survey-results.md

```diff
@@ -0,0 +1,12 @@
+---
+category: news
+by: Julien Richard-Foy, Scala Center
+title: "Results of the 2019 Scala Developer Survey"
+---
+
+You have been about 4 thousands Scala developers to fill the survey and we are grateful
+to you! The survey results help us identify the trends in how you use the language in
```

Comment:
The first sentence reads awkwardly to a native speaker, and there isn't a spot fix; it needs rearranging. Perhaps simply:

> Four thousand Scala developers filled out the survey. We are grateful to you all!

I would also suggest a paragraph break at this point.